filebeat/input/log/input.go

```diff
@@ -558,6 +568,8 @@ func (p *Input) scan() {
 				continue
 			}
 			if err != nil {
+				p.healthy = false
+				p.updateStatus(status.Degraded, fmt.Sprintf(harvesterErrMsg, newState.Source, err))
```

Comment:
I'm not quite sure the exact contents of this error, but to report it to FleetUI we could make sure there is some context like the path of the file that cannot be harvested.

Reply:
@belimawr The error will be in following format:

`Harvester could not be started on new file: %s, Err: %s`

It should be fine as it contains the new file info and the error info.